.github/workflows/docker_ci.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           RABBITMQ_DEFAULT_USER: guest
           RABBITMQ_DEFAULT_PASS: guest
       ewh:
-        image: docker.pkg.github.com/sanger/event_warehouse/event_warehouse:master
+        image: docker.pkg.github.com/sanger/event_warehouse/event_warehouse:development
```

Comment:
Don't we want this as master?

Reply:
yes, but it wont work until is merged into master